test/lib/next-modes/next-start.ts

```diff
@@ -53,6 +50,10 @@ export class NextStartInstance extends NextInstance {
         __NEXT_TEST_MODE: 'e2e',
       },
     }
+
+    if (this.childProcess) {
+      throw new Error('next already started')
```

Comment:
Why this change?

Reply:
didn't fully reverted everything, reverted now